reservation/reservation.go

```diff
@@ -12,10 +12,13 @@ import (
 // Reservation is a type that represents a lock on a resource. At most one reservation
 // can exist for an individual resource at any time.
 type Reservation struct {
-	stopped    bool
-	key, value string
-	getConn    func() redis.Conn
-	ttl        time.Duration
+	stopped bool
+	key     string
```

Comment:
No, it's currently not used for logging anywhere so I'm going to leave it private.